scripts/samplesheet_generator.py

```diff
@@ -413,7 +523,7 @@ def main(lims, args):
     else:
         process = Process(lims, id=args.pid)
 
-        if process.type.name == 'Load to Flowcell (NovaSeq 6000 v2.0)':
+        if "Load to Flowcell (NovaSeq 6000 v2.0)" in process.type.name:
```

Comment:
```suggestion
        if "Load to Flowcell (NovaSeq 6000 v2.0)" == process.type.name:
```

In order to be exactly corresponding to previous version of the code.